devtools-integration-tests/testerina-tests/services-test/tests/main_test.bal

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/test;
+import ballerina/log;
+import ballerina/io;
+
+service helloTest on new http:Listener(9092) {
+    resource function sayHello(http:Caller caller, http:Request req) {
+        var result = caller->respond("Hello, Test!");
+        if (result is error) {
+            log:printError("Error sending response", result);
+        }
+    }
+}
+
+# Test service
+@test:Config {
+
+}
+function testFunction() {
```

Comment:
I think similar test cases are already added at https://github.com/ballerina-platform/ballerina-distribution/blob/master/stdlib-integration-tests/http/tests/http_1_1_passthrough_test.bal

We can add test cases that use mocking with services. WDYT?

Reply:
Yes. Can add test cases for Object mocking that involves the http module